.gitignore

```diff
@@ -1,16 +1,24 @@
-_build/
-cover/
-deps/
+# The directory Mix will write compiled artifacts to.
+/_build/
+
+# If you run "mix test --cover", coverage assets end up here.
+/cover/
+
+# The directory Mix downloads your dependencies sources to.
+/deps/
+
+# Where third-party dependencies like ExDoc output generated docs.
+/doc/
+
+# Ignore .fetch files in case you like to edit your project deps locally.
+/.fetch
+
+# If the VM crashes, it generates a dump, let's ignore it too.
 erl_crash.dump
+
+# Also ignore archive artifacts (built via "mix archive.build").
 *.ez
-*.eqc
-current_counterexample
-.eqc-info
-rel/
-.tool-versions-e
-.elixir_ls/
```

Comment:
Don't remove those.

Reply:
I think Mike added the one form core and I added the one generated by `mix new`, I'll return the dot files, but things like `rel` and `eqc` related, we don't need (yet).